lib/rides/widgets/search_ride_filter.dart

```diff
@@ -209,7 +214,7 @@ class SearchSuggestionFilter {
               inputFormatters: <TextInputFormatter>[FilteringTextInputFormatter.digitsOnly],
             ),
           ),
-          Text(S.of(context).searchSuggestionsFilterDeviationHours),
+          Text(wholeDay ? S.of(context).searchRideFilterDeviationDays : S.of(context).searchRideFilterDeviationHours),
```

Comment:
this does not seem to work correctly, I had switched off wholeDay and it still shows days